CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to cofidectl
+
+Linting, unit tests and integration tests are run by GitHub Actions against changes proposed to `cofidectl`.
+These checks may also be executed locally.
+
+## Running linters
+
+To run the `golangci-lint` checks:
+
+```sh
+just lint
```

Comment:
Remarkably, it's not just `just lint` - one needs to set up the correct version of `golangci-lint` (v1, compiled with the same go version as this project). I did that manually but I wonder if there is a good pattern that could be applied to save this effort for others.